mgmt-lambda/handlers/updateHandler.ts

```diff
@@ -156,5 +156,5 @@ async function checkIfLambdaFunctionWithNameExists(client: LambdaClient, functio
   const command = new GetFunctionCommand({ FunctionName: functionName })
   const result: GetFunctionCommandOutput = await client.send(command)
 
-  return typeof result?.Configuration?.FunctionArn !== 'undefined'
+  return typeof result?.Configuration?.FunctionArn !== 'undefined' && result?.Configuration?.FunctionArn !== null
```

Comment:
why not replace this whole line with `return result?.Configuration?.FunctionArn != null`. It is shorter. Any particular reason?